packages/react/src/internal/useId.js

```diff
@@ -65,7 +65,7 @@ export function useId(prefix = 'id') {
 
 /**
  * Generate a unique id if a given `id` is not provided
- * @param {string} id
+ * @param {string|undefined} id
```

Comment:
`useId` signature says it returns a string, under what circumstances would this return undefined?

Reply:
@francinelucca the change pertains to function `id` argument type, not the return type.

Reply:
my bad! I read that wrong